packages/e2e/features/ui/components/storage/storage-browser/file-preview.feature

```diff
@@ -0,0 +1,77 @@
+Feature: File Preview with Storage Browser
+
+  Background:
+    Given I'm running the example "ui/components/storage/storage-browser/default-auth"
+    And I type my "email" with status "CONFIRMED"
+    And I type my password
+    And I click the "Sign in" button
+
+  @react
+  Scenario: Open and close file preview for text file
+    When I click the first button containing "public"
+    Then I click the button with label "001_dont_delete_text_file_preview_end_to_end_testing.txt file"
+    Then I see "File Preview"
+    Then I see "File Information"
+    Then I see "001_dont_delete_text_file_preview_end_to_end_testing.txt"
```

Comment:
this is redundant.
it just tests for text in the document, which you tested for already 3 lines up

```
Then I click the button with label "001_dont_delete_text_file_preview_end_to_end_testing.txt file"
```

Alternatively it needs adjusting to test for the text in the preview container.